pysnmp/proto/mpmod/rfc3412.py

```diff
@@ -523,6 +526,9 @@ def prepare_response_message(
         elif securityLevel == 3:
             msgFlags |= 0x03
         else:
+            if securityModel in snmpEngine.security_models:
```

Comment:
`def prepare_data_elements` in `rfc3412.py` already blocked invalid msgFlags from being passed on, so it is impossible to have a `securityLevel` value here.